mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/domain/course/LessonContentLesson.java

```diff
@@ -0,0 +1,43 @@
+package org.motechproject.nms.mobileacademy.domain.course;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+/**
+ * Contains details about actual content files to be played while playing a lesson.
+ */
+@Entity
+public class LessonContentLesson {
+
```

Comment:
We are not pushing for a spec-change to the MA response schema at this time. Given that assumption, I have the option to map their schema as-is (as close as possible) to our domain model or diverge.

I figured that it would be better to diverge and change the API spec at the same time than just 1 and have a hard time during debugging, trying to map theirs to ours. Happy to change it though..